_component_design/text-input.md

```diff
@@ -11,7 +11,7 @@ status: stable
 last-modified: 2018-10-26
 helix-ui-css: true
 helix-ui-javascript: na
-pagelink: https://rackerlabs.github.io/helix-ui/components/text-inputs/
+pagelink: components/text-input/
```

Comment:
also relevant: `components/textarea/`

Reply:
yep.  will wait until we split textarea design docs out from text-input